contributing.md out of date

# Bug report

## Describe the bug

I wanted to contribute something but I have no idea how to develop next.js, contributing.md seems out of date as when I run

```
npm run bootstrap
npm run dev
yarn testonly
```

Then I get lot of errors "SyntaxError: Unexpected identifier" of regeneratorRuntime.

Could you tell how I can properly start developing next.js (both next and next-server) and test it before sending PR?

@timneutkens I am experiencing similar issues. 

```bash
npm i
npm run bootstrap
npm run dev
```

If I run the above steps, I run into the following error:

![image](https://user-images.githubusercontent.com/1099139/46631270-ca092880-cafb-11e8-9645-bdf9841e4bf8.png)

Tests are also failing with `Cannot find module '@babel/core'` (although I had them passing at one point...)

That said, I still can't get it to run with a local project. I end up with a series of errors when I hit localhost:3000, e.g.

```
{ Error: Cannot find module 'next-server/dist/lib/utils'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.next-server/dist/lib/utils (/Users/206476943/git/temp/next-test/.next/server/static/development/pages/_app.js:463:18)
    at __webpack_require__ (/Users/206476943/git/temp/next-test/.next/server/static/development/pages/_app.js:23:31)
    at Object.../next.js/packages/next/dist/pages/_app.js (/Users/206476943/git/temp/next-test/.next/server/static/development/pages/_app.js:138:14)
    at __webpack_require__ (/Users/206476943/git/temp/next-test/.next/server/static/development/pages/_app.js:23:31)
    at Object.0 (/Users/206476943/git/temp/next-test/.next/server/static/development/pages/_app.js:330:18)
    at __webpack_require__ (/Users/206476943/git/temp/next-test/.next/server/static/development/pages/_app.js:23:31)
    at /Users/206476943/git/temp/next-test/.next/server/static/development/pages/_app.js:91:18
    at Object.<anonymous> (/Users/206476943/git/temp/next-test/.next/server/static/development/pages/_app.js:94:10)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12) code: 'MODULE_NOT_FOUND' }
{ Error: Cannot find module 'next-server/dist/lib/utils'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
...
```

--- a/contributing.md
+++ b/contributing.md
@@ -7,7 +7,7 @@ Our Commitment to Open Source can be found [here](https://zeit.co/blog/oss)
 3. Run `npm run bootstrap`, which will link all repositories locally
 4. Run `npm run dev` to build and watch for code changes
 
-### To run tests
+## To run tests
 
 Running all tests:
 
@@ -27,26 +27,30 @@ Running just one test in the `production` test suite:
 yarn testonly --testPathPattern "production" -t "should allow etag header support"
 ```
 
-### Running the integration test apps without running tests
+## Running the integration test apps without running tests
 
 ```
 ./node_modules/.bin/next ./test/integration/basic
 ```
 
-### Testing in your own app
+## Testing in your own app
 
-You only have to link the package once:
-
-```
-cd packages/next
-```
+First `next-server` needs to be linked:
 
 ```
+cd packages/next-server
 npm link
 ```
 
+Then `next` needs to link to `next-server`, and be linked itself:
 
-Then link the `next` package inside your app:
+```
+cd packages/next
+npm link
+npm link next-server
+```
+
+And finally, link the `next` package inside your app:
 
 ```
 npm link next

